src/Kestrel.Core/Internal/Infrastructure/HttpUtilities.cs

```diff
@@ -177,6 +178,87 @@ internal static unsafe HttpMethod GetKnownMethod(byte* data, int length, out int
             return HttpMethod.Custom;
         }
 
+        /// <summary>
+        /// Parses string <paramref name="value"/> for a known HTTP method.
+        /// </summary>
+        /// <remarks>
+        /// A "known HTTP method" can be an HTTP method name defined in the HTTP/1.1 RFC.
+        /// The Known Methods (CONNECT, DELETE, GET, HEAD, PATCH, POST, PUT, OPTIONS, TRACE)
+        /// </remarks>
+        /// <returns><c>true</c> if the input matches a known string, <c>false</c> otherwise.</returns>
+        public static string GetKnownMethod(string value, out HttpMethod method)
```

Comment:
This method is taking string `value` and then returning it back again. Why not just return HttpMethod instead of an out param?

Also XML docs need to be updated.

Reply:
Done